nixos/modules/system/boot/luksroot.nix

```diff
@@ -889,6 +942,10 @@ in
           message = "boot.initrd.luks.devices.<name>.bypassWorkqueues is not supported for kernels older than 5.9";
         }
 
+        { assertion = all (dev: dev.tryEmptyPassphrase && dev.keyFileTimeout != null ) (attrValues luks.devices);
+          message = "boot.initrd.luks.devices.<name>.tryEmptyPassphrase requires a keyFileTimeout to be set";
+        }
+
```

Comment:
This assertion is still wrong. It says that *all* devices *must* have `tryEmptyPassphrase` and `keyFileTimeout`. You want `->`, not `&&`.

Reply:
Also is this assertion even true? `keyFileTimeout` isn't required with `tryEmptyPassphrase` with systemd stage 1, and it's not used at all in the regular initrd in this PR.

Reply:
yeah this is wrong, let me fix this

Reply:
Yeah I'm not sure why I had this in the first place. I don't think this is necessary.